db/migrate/20230119042016_add_users_ref_to_like.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ] Again, the `user` model is referenced using foreign key in models as `AuthorId`, and `author` an alias of `user` hence, our migration query should be similar to:

```ruby
...
  def change
    add_reference :likes, :author, references: :users, null: false, foreign_key: {to_table: :users}
  end
...

```

or.. 

```ruby
  create_table :likes do |t|
    . . .
    t.references :author, foreign_key: { to_table: 'users' } 
    . . .
  end
```

Reply:
well resolved! ✔️